meshtastic/tests/test_api_baseline_comparison.py

```
@@ -0,0 +1,627 @@
+def load_baseline() -> dict[str, Any] | None:
+    """Load the stored baseline from disk.
+
+    Returns None if no baseline exists.
+    Checks for master baseline first (used in CI comparisons),
+    falls back to standard baseline file.
+    """
+    baseline_file = get_baseline_file()
+    if not baseline_file.exists():
+        return None
+
+    with open(baseline_file, "r", encoding="utf-8") as f:
```

Ruff (0.15.7)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)